src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(health) {
+    const FULL_HEALTH = 100;
+
+    this.health = health;
+
+    this.health = FULL_HEALTH;
+    Animal.alive.push(this);
+  }
+  static animals() {
```

Comment:
This method is not needed, because you already handle this in `bite()` method of `Carnivore` class